lua/harpoon/config.lua

```diff
@@ -117,6 +117,10 @@ function M.get_default_config()
                     set_position = true
                     bufnr = vim.fn.bufnr(list_item.value, true)
                 end
+                if not vim.api.nvim_buf_is_loaded(bufnr) then
```

Comment:
Buffers created by `bufnr` are always unloaded and unlisted. We need to check if it's loaded, and if not then load it and set buflisted.

Reply:
Just tested, fix #377